packages/fast-development-site-react/src/components/site/configuration-panel.tsx

```diff
@@ -57,10 +68,11 @@ const style: ComponentStyles<ConfigurationPanelManagedClasses, DevSiteDesignSyst
             padding: `${toPx(6)} ${toPx(8)}`,
```

Comment:
nit: none of these need to be `toPx()`

Reply:
I think we should make a seperate task to clean these up. I removed them all from Fast Form but since this package will be rebuilt soon I figured I would save that work for that task.